packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageGroup.tests.js

```diff
@@ -287,6 +289,23 @@ QUnit.module('MessageGroup', moduleConfig, () => {
                 assert.strictEqual(this.invalidateStub.callCount, 1);
             });
         });
+
+        QUnit.test('messageTemplate should be called on bubble template call', function(assert) {
+            const messageTemplate = sinon.stub();
+            const message = {
+                text: 'CustomText',
+                timestamp: 1234567,
+                author: { name: 'someName', id: 'someId' },
+            };
+
+            this.reinit({
+                items: [message],
+                messageTemplate,
+            });
+
+            assert.strictEqual(messageTemplate.callCount, 1, 'messageTemplate function was called on bubble template render');
+            assert.deepEqual(messageTemplate.lastCall.args[0], message, 'messageTemplate function was called with correct data');
+        });
```

Comment:
It's strange for me that we check callCount here instead of usual check that correct option is passed to the bubble. But I gonna trust you about this